### PR TITLE
12.0 mig partner multi relation

### PR DESCRIPTION
**Pull request by ddufresne**

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]

File: partner_multi_relation/views/res_partner.xml
Comment (nikul-serpentcs) [inline partner_multi_relation/views/res_partner.xml]: @ddufresne Remove  `<data>`, not need

File: partner_multi_relation/tests/__init__.py
Comment (nikul-serpentcs) [inline partner_multi_relation/tests/__init__.py]: @ddufresne Remove Copyright from `init` file

**Review by nikul-serpentcs — COMMENTED** [review]
Improve Code

**Review by NL66278 — APPROVED** [review]
:+1:  LGTM

File: partner_multi_relation/README.rst
Comment (chienandalu) [inline partner_multi_relation/README.rst]: Shouldn't the images hang from `static`? otherwise the seem to be not reachable?

**Review by chienandalu — CHANGES_REQUESTED** [review]
Just a little issue with the readme images. Otherwise ok :+1: Thanks!

File: partner_multi_relation/readme/USAGE.rst
Comment (tarteo) [inline partner_multi_relation/readme/USAGE.rst:1]: https://github.com/OCA/maintainer-tools/blob/master/template/module/readme/USAGE.rst
"it MUST NOT contain reStructuredText sections only body text (paragraphs, lists, tables, etc)"

I'm not sure what the consequences are when you use sections in the usage, description etc. But is says here that you must not do it :S

Comment (chienandalu) [reply partner_multi_relation/readme/USAGE.rst:1]: Well, it depends on the paragraph level hierarchy. It would fail in CI if it wasn't correct

**Review by tarteo — APPROVED** [review]

File: partner_multi_relation/readme/USAGE.rst
Comment (pedrobaeza) [reply partner_multi_relation/readme/USAGE.rst:1]: The only level allowed here is `~~~~~~`

Comment (tarteo) [reply partner_multi_relation/readme/USAGE.rst]: Thanks!

Comment (tarteo) [reply partner_multi_relation/readme/USAGE.rst:1]: Oh so it must be changed?

Comment (pedrobaeza) [reply partner_multi_relation/readme/USAGE.rst:1]: I'm checking and it seems it's correctly rendered... @sbidoul `-----` titles are tolerated inside this section?

Comment (sbidoul) [reply partner_multi_relation/readme/USAGE.rst:1]: `~` is accepted yes, other may work or break anytime.

Comment (pedrobaeza) [reply partner_multi_relation/readme/USAGE.rst:1]: @ddufresne can you please switch to this title style for avoiding problems?

File: partner_multi_relation/README.rst
Comment (chienandalu) [reply partner_multi_relation/README.rst]: image

Comment (pedrobaeza) [reply partner_multi_relation/README.rst]: I'm able to see them:

seleccion_003

But the problem is that there's no good path for being displayed everywhere: apps store, README, Odoo viewer...

Comment (chienandalu) [reply partner_multi_relation/README.rst]: My screenshot was taken in the runbot instance app form wich seems to block any static resource linked outside the static folder.

Comment (pedrobaeza) [reply partner_multi_relation/README.rst]: It's not only that, but the path: although you put it inside static folder, you have to add the module name prefix, which screws then the visualization of the README on GitHub. I remember an issue about this, but I'm not able to find it. @elicoidal can you refresh my memory?

**Review by chienandalu — APPROVED** [review]

**Comment by pedrobaeza** [comment]
@ddufresne please change title underline and for the images, there's a trick we can use: put the absolute URL that GitHub will give you when this got merged, and they will be shown properly on all the places.

You have some examples in mis-builder README: https://github.com/OCA/mis-builder/blob/10.0/mis_builder/README.rst

**Comment by ddufresne** [comment]
@pedrobaeza I fixed the title underline and used your trick for images.

File: partner_multi_relation/tests/__init__.py
Comment (ddufresne) [reply partner_multi_relation/tests/__init__.py]: @nikul-serpentcs sorry, won't do that. I am not sure about OCA's policy about that and otherwise, I am not confortable removing someone's copyright from a python file.

Even though, I should not be the person commiting such change. Also, this should not block the PR.

File: partner_multi_relation/views/res_partner.xml
Comment (ddufresne) [reply partner_multi_relation/views/res_partner.xml]: fixed

File: partner_multi_relation/tests/__init__.py
Comment (pedrobaeza) [reply partner_multi_relation/tests/__init__.py]: Well, it was a mistake to put copyright in a file that doesn't contain code, but it's not blocking either.

Comment (NL66278) [reply partner_multi_relation/tests/__init__.py]: We followed older guidelines to put copyright in ALL python files. I have no problem with removing it here and from any other ```__init__.py``` file.

**Comment by pedrobaeza** [comment]
I'm going to squash some commits, amend copyright things after the grant of the author, and merge this.

**Comment by pedrobaeza** [comment]
Thanks to all for reviewing and specially to @ddufresne for extending the README. Merging!

**Pull request merged**
